script/configure-web-proxy

```diff
@@ -39,20 +39,32 @@ done
 
 echo -e "[openid]\nhttpsonly = 0" > /etc/openqa/openqa.ini.d/01-enable-http-auth.ini
 
+etc_proxy=/etc/${web_proxy/apache/apache2}
+for v in vhosts.d sites-available; do
+    if [[ -e "${etc_proxy}/${v}/openqa.conf.template" ]]; then
+        vhosts_dir="${etc_proxy}/${v}"
```

Comment:
```suggestion
        vhosts_dir="${etc_proxy}/${v}"
        break
```

Reply:
Ah, I see there's a "Commit Suggestion" button -- never mind, did that by hand locally instead :-)